### PR TITLE
Add save for save active federation creation block height

### PR DESCRIPTION
**Pull request by jeremy-then**
Add save for save active federation creation block height

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java [hunk 1]
```diff
@@ -948,6 +948,30 @@ void save_savePendingFederation_preRSKIP123_shouldBeSavedInStorageSerializedFrom
 
     }
 
+    @Test
+    void saveActiveFederationCreationBlockHeight_postIrisAndHeightIsNotNull_shouldSaveToStorage() {
```
Comment (nathanieliov) [inline rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java:952]:
```suggestion
    void saveActiveFederationCreationBlockHeight_whenIrisIsActiveAndHeightIsNotNull_shouldSaveToStorage() {
```

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java [hunk 2]
```diff
@@ -948,6 +948,30 @@ void save_savePendingFederation_preRSKIP123_shouldBeSavedInStorageSerializedFrom
...
+        // Arrange
+
+        ActivationConfig.ForBlock irisActivation = ActivationConfigsForTest.iris300().forBlock(0L);
```
Comment (nathanieliov) [inline rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java:955]:
```suggestion
        ActivationConfig.ForBlock irisActivations = ActivationConfigsForTest.iris300().forBlock(0L);
```

**Review by nathanieliov — APPROVED** [review]
Just two small suggestions

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java (hunk 1 above)
Comment (jeremy-then) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java:893]: Already discussed in the poll.

File: rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java (hunk 2 above)
Comment (jeremy-then) [reply rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java]: Updated.

**Review by wilmerrootstock — APPROVED** [review]
LGTM!

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**